ui/src/config/invenioConfig.js

```diff
@@ -30,6 +30,12 @@ export const invenioConfig = {
       { value: 'PERIODICAL_ISSUE', text: 'Periodical issue' },
     ],
   },
+  documentRequests: {
+    physicalItemProviders: {
+      acq: { name: 'Acquisition', pid_type: 'Order' },
+      ill: { name: 'InterLibrary', pid_type: 'BorrowingRequest' },
```

Comment:
Shouldn't we use `acqoid` and  `illbid` for the `pid_type` here?

Reply:
👍 I agree